### PR TITLE
[WIP][local-up-cluster] Adjust oom score for coredns

### PR DESCRIPTION
**Pull request by dims**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by aojea** [comment]
can we dump the existing `/etc/resolv.conf ` ??

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -742,12 +742,64 @@ function wait_coredns_available(){
     exit 1
   fi
 
+  echo "================= BEGIN DEBUG ========================"
+  echo "====== set overcommit_memory ====="
+  echo 1 | sudo tee /proc/sys/vm/overcommit_memory
+  sudo cat /proc/sys/vm/overcommit_memory
+  echo "====== ps ====="
```
Comment (aojea) [inline hack/local-up-cluster.sh:749]:
```suggestion
  echo "====== resolv.conf ====="
  sudo cat /etc/resolv.conf
  echo "====== ps ====="
```

**Comment by dims** [comment]
stopping this for now. will pick it up later.

**Pull request closed without merging**